linters.js

```diff
@@ -0,0 +1,58 @@
+/* Results on Node 19.2.0, GitHub Actions:
+
+PostCSS sync:   74 ms  (1.1 times faster)
+PostCSS:        78 ms
+LibSass sync:   110 ms (1.4 times slower)
+LibSass:        112 ms (1.4 times slower)
+Less:           140 ms (1.8 times slower)
+Dart Sass sync: 197 ms (2.5 times slower)
+Dart Sass:      376 ms (4.8 times slower)
+*/
```

Comment:
Running in CI to get correct values : https://github.com/romainmenke/postcss-benchmark/actions/runs/5436306980/jobs/9886062225?pr=1